agent/php_txn.c

```diff
@@ -669,6 +669,58 @@ static void nr_php_txn_send_metrics_once(nrtxn_t* txn TSRMLS_DC) {
 #undef FMT_BOOL
 }
 
+void nr_php_txn_create_agent_version_metric(nrtxn_t* txn, const char* version) {
+  char* metric_name = NULL;
+
+  if (NULL == txn) {
+    return;
+  }
+
+  if (nr_strempty(version)) {
+    return;
+  }
+
+  metric_name = nr_formatf("Supportability/PHP/AgentVersion/%s", version);
```

Comment:
since we are just appending and not putting version in the middle of the string, can't we just use nr_strcat?

Reply:
Switching to `nr_strcat` would still require measuring the string lengths and allocating a buffer.  There may be some improvement in speed - but these `sprintf` ops take 10s of microseconds so it is not going to be noticeable in latency or CPU since it occurs once per request and after the PHP app is done.